packages/analysis-report/README.md

```diff
@@ -0,0 +1,585 @@
+
+Individual result item that displays a single analysis outcome with interactive elements.
+
+#### Props
+```typescript
+interface AnalysisResultProps {
+  // Content
+  text: string;             // HTML string with analysis feedback
+  score: number;            // 0-100 score
+  rating: "good" | "OK" | "bad" | "feedback" | "upsell";
+  id: string;              // Unique identifier
+  
+  // Marking functionality
+  hasMarks: boolean;       // Whether this result supports text marking
+  marker?: Marker;         // Marking instructions
+  marksButtonStatus?: "enabled" | "disabled" | "hidden";
+  
+  // Edit functionality
+  hasEditButton: boolean;  // Whether to show edit button
+  editFieldName?: string;  // Field to focus when edit clicked
+  
+  // Styling
+  bulletColor: string;     // Color for score indicator
+  marksButtonClassName?: string;
+  editButtonClassName?: string;
+  
+  // Accessibility
+  ariaLabelMarks: string;
+  ariaLabelEdit?: string;
+  
+  // State
+  pressed: boolean;        // Whether marks are active
+  suppressedText?: boolean;// Grey out the text
+}
+```
+
+#### Features
+- Sanitizes HTML in text content
+- Manages button states and interactions
+- Handles accessibility attributes
+- Supports custom styling
+
+### SiteSEOReport
+
+Displays aggregate SEO scores with a stacked progress bar visualization.
+
+#### Props
+```typescript
+interface SiteSEOReportProps {
+  className?: string;
+  seoAssessmentText: string;
+  seoAssessmentItems: Array<{
+    html: string;          // HTML content for item description
+    value: number;         // Percentage value (0-100)
+    color: string;         // Color for progress bar segment
+  }>;
+  barHeight?: string;      // Custom height for progress bar
+}
+```
+
+#### Features
+- Renders stacked progress bars
+- Displays score breakdowns
+- Supports custom styling and dimensions
+- Handles responsive layouts
+
+## Component Interactions
+
+### Data Flow
+```mermaid
+sequenceDiagram
+    participant P as Parent App
+    participant C as ContentAnalysis
+    participant A as AnalysisResult
+    participant M as Marker System
+    
+    P->>C: Analysis Results
+    C->>A: Individual Result
+    A->>M: Marking Request
+    M->>P: Mark Positions
+    P->>M: Apply Marks
+    M->>A: Update UI
+```
+
+### State Management
+
+The components follow a unidirectional data flow:
+
+1. **Parent Level**
+   ```javascript
+   {
+     activeMarker: string | null;    // Currently active marker
+     marks: Mark[];                  // Active text marks
+     results: AnalysisResult[];      // Analysis results
+   }
+   ```
+
+2. **ContentAnalysis Level**
+   ```javascript
+   {
+     collapsibleStates: {           // Section collapse states
+       [category: string]: boolean
+     }
+   }
+   ```
+
+3. **AnalysisResult Level**
+   ```javascript
+   {
+     isHovered: boolean;            // UI hover state
+     isPressed: boolean;            // Marker active state
+   }
+   ```
```

Comment:
I would remove these parts as they mostly repeat information from above.

Reply:
I'd definitely keep the sequence diagram, that's nice to have. This last State Management section I don't really understand, but seems different from what I read before as well.